projects/keycloak/build.sh

```diff
@@ -18,24 +18,71 @@
 # Retrieve JDK-17
 wget https://download.java.net/openjdk/jdk17/ri/openjdk-17+35_linux-x64_bin.tar.gz
 tar -zxf openjdk-17+35_linux-x64_bin.tar.gz
-JAVA_HOME=$SRC/keycloak/jdk-17
+cp -r jdk-17 $OUT/
+JAVA_HOME=$OUT/jdk-17
+PATH=$JAVA_HOME/bin:$PATH
 
 # Build Keycloak
+
+## Maven build arguments
 MAVEN_ARGS="-Djavac.src.version=17 -Djavac.target.version=17 "
 MAVEN_ARGS=$MAVEN_ARGS"-DskipTests -Dgpg.skip -Dmaven.source.skip "
-MAVEN_ARGS=$MAVEN_ARGS"-DskipExamples -DskipTestsuite"
-$MVN clean package $MAVEN_ARGS
+MAVEN_ARGS=$MAVEN_ARGS"-DskipExamples -DskipTestsuite -DskipQuarkus"
+
+## Exclude unfuzzed modules
```

Comment:
Can you add a comment why the build script does this?

Reply:
done.